openapi_python_client/schema/openapi_schema_pydantic/callback.py

```diff
@@ -0,0 +1,19 @@
+from typing import Dict
+
+Callback = Dict[str, "PathItem"]  # type: ignore [name-defined]
```

Comment:
The vendored lib have some typing issues, neither flak8 or mypy is ok with this ~Lazy type definition. 
It seems to be used here because it is not possible to import the related model due to circular dependencies.

I've ask David, he told me it was good enought to ignore that specific error  here;
-  with a comment for mypy 
- with a custom configuration in the `.flak8` file for flak8

This errors occurs twice, in that file and in the `encoding.py`

Reply:
Can we just import `PathItem` here?

Reply:
Unfortunatly not that easy: 
```
ImportError: cannot import name 'Callback' from partially initialized module 'openapi_python_client.schema.openapi_schema_pydantic.callback' (most likely due to a circular import) (/home/xxxx/openapi-5GC/openapi-python-client/openapi_python_client/schema/openapi_schema_pydantic/callback.py)
```

It will need a deeper refactor of the lib

Reply:
Oh if it's a circular import than you can import just for type checking like:

```python
from typing import TYPE_CHECKING

if TYPE_CHECKING
    from .whatever import PathItem
```

Then it will resolve for mypy so type checking can happen but not mess up Python with the circular import.

Reply:
Nice, I wasn't aware of it. resolved on https://github.com/triaxtec/openapi-python-client/pull/426/commits/6b734c691825e2ff4f2fca92025a51f197df62ff